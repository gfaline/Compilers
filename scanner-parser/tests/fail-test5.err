Fatal error: exception Failure("illegal name _b")
