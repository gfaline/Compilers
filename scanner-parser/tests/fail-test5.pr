
objdef Student
{
  int id;
  str name;
}


 Student stu;
 stu.id = 25111;
 stu.name = 'John';
 fn celebrate(obj o)
 {
   if o.name == 'John'
   {
     print('Name..')
  }
}

bind(stu.name, celebrate(stu));