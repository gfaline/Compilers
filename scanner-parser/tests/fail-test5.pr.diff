diff: fail-test5.pr.err: No such file or directory
diff: testss/tests/fail-test5.pr.err: No such file or directory
