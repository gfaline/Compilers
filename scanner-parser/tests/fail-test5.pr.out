tests/fail-test5.pr
